src/js/tests/strings.rs

```diff
@@ -0,0 +1,16 @@
+use super::super::*;
+use crate::assert_js;
```

Comment:
Can we just have a `tests.rs` module to match the other modules?

Reply:
I'd rather break it up. I got to 800+ lines on #1027 and I'm sure I was quite a long way short of done.

I wasn't able to work through the erl tests, because there are different edge cases you need to consider and things like named arguments behave very differently. Currently I'm breaking it up into files per language feature, matching the chapters of https://gleam.run/book/tour/

Reply:
OK 👍